File: features/second/src/main/java/com/inquallity/daggersandbox/second/SecondActivity.java
package com.inquallity.daggersandbox.second;

import android.os.Bundle;
import android.util.Log;

import androidx.appcompat.app.AppCompatActivity;

import com.inquallity.daggersandbox.feature.second.R;

public class SecondActivity extends AppCompatActivity {

    public SecondActivity() {
        super(R.layout.ac_second);
    }

    @Override
    public void onCreate(Bundle bundle) {
        super.onCreate(bundle);
        Log.d("OYAEBU", "Second activity started...");
    }
}

File: features/second/src/main/java/com/inquallity/daggersandbox/second/SecondActivity.kt
package com.inquallity.daggersandbox.second

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
import com.inquallity.daggersandbox.feature.second.R

class SecondActivity : AppCompatActivity(R.layout.ac_second) {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Log.d("OYAEBU", "Second activity started...")
    }
}